tests/unit/ui/components/TestUserAvatar.js

```diff
@@ -22,30 +22,11 @@ describe(__filename, () => {
     const user = getCurrentUser(state.users);
     const root = renderUserAvatar({ user });
 
-    expect(root.find('img.UserAvatar')).toHaveLength(1);
-    expect(root.find('.UserAvatar')).toHaveProp(
+    expect(root.find('.UserAvatar-image')).toHaveLength(1);
+    expect(root.find('.UserAvatar-image')).toHaveProp(
       'src', 'http://tofumatt.com/photo.jpg');
   });
 
-  it('renders extra classNames when rendering an <img> tag', () => {
```

Comment:
Oops, only one of these should have been removed. We still want to make sure our custom `className` is rendered, so you'll want a test that ensures `<UserAvatar className="foo" />` will pass `expect(root.find('.foo')).toHaveLength(1)`